lib/full-site-editing/template-loader.php

```diff
@@ -291,3 +300,34 @@ function gutenberg_get_edit_template_link( $link, $post_id ) {
 	return admin_url( sprintf( $edit_link, urlencode( $template->id ), $template->type ) );
 }
 add_filter( 'get_edit_post_link', 'gutenberg_get_edit_template_link', 10, 2 );
+
+/**
+ * Checks the HTML to be rendered for any emojis.
+ * If there are none, then the emoji script is removed from the `<head>`.
+ *
+ * @param string $html The HTML to be rendered.
+ *
+ * @return string The HTML.
+ */
+function _gutenberg_maybe_remove_emoji_detection_script( $html ) {
+	$multi3  = '(?:[\xE0-\xEF]..)';
+	$multi4  = '(?:[\xF0-\xF4]...)';
```

Comment:
Is there some solid reference on which we can base this?

Reply:
Removed the regex so I'm marking this as resolved 👍